packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -9,28 +11,36 @@ import { RawTimeRange, TimeZone } from './time';
 
 export enum PluginExtensionTypes {
   link = 'link',
+  component = 'component',
 }
 
-export type PluginExtension = {
+export type PluginExtensionBase = {
```

Comment:
I guess this is a breaking change since we use to export `PluginExtension` as the base class and didn't really have anything like the new `PluginExtension` type.

Reply:
On the other hand. We could probably have a look at incident and ML to see if there are any usages of this type.

Reply:
Another question, do we need to export the `PluginExtensionBase` ?

Reply:
Yepp, good point, I will look in the Incident and ML apps if they are using this in any way. And probably we don't need to expose the `PluginExtensionBase` indeed!

Reply:
Looks like the [grafana-ml-app](https://github.com/grafana/machine-learning/blob/530e493272c24f00600db4f3a86edb4f2724de87/ui-plugins/grafana-ml-app/src/module.ts#L4) and the [grafana-incident-app](https://github.com/grafana/incident/blob/940e54e6d1c6cf40f4b6b2183e72d2a8db4afb1d/packages/grafana-incident-app/src/module.tsx#L3) are not depending on the `PluginExtension`, so we are good from this perspective.

Stopped exporting `PluginExtensionBase` in https://github.com/grafana/grafana/pull/68600/commits/8ee09cdd9a0b7309d8cc5dd0722a8720a64b9c4d.